doc/man3/SSL_CTX_set_verify.pod

```diff
@@ -144,6 +144,9 @@ B<Client mode:> ignored (see BUGS)
 
 If the B<mode> is SSL_VERIFY_NONE none of the other flags may be set.
 
+If the B<mode> is not modified by C<SSL_CTX_set_verify()> or
+C<SSL_set_verify()>, the behavior is similar to SSL_VERIFY_NONE.
+
```

Comment:
Not just similar. I.e., `If the B<mode> is not modified ..., it defaults to B<SSL_VERIFY_NONE>`.

Reply:
Reworded, let me know if the new version sounds better